GW2EIEvtcParser/ParsedData/FightData.cs

```diff
@@ -295,10 +302,11 @@ static internal FightLogic DetectFight(int id, AgentData agentData, EvtcParserSe
 
     internal void CompleteFightName(CombatData combatData, AgentData agentData)
     {
+        FightNameNoMode = Logic.GetLogicName(combatData, agentData);
         FightName = Logic.GetLogicName(combatData, agentData)
-            + (_encounterMode == EncounterMode.CM ? " CM" : "")
-            + (_encounterMode == EncounterMode.LegendaryCM ? " LCM" : "")
-            + (_encounterMode == EncounterMode.Story ? " Story" : "")
+            + (FightMode == EncounterMode.CM || FightMode == EncounterMode.StoryCM ? " CM" : "")
+            + (FightMode == EncounterMode.LegendaryCM ? " LCM" : "")
+            + (FightMode == EncounterMode.Story ? " Story" : "")
```

Comment:
StoryCM should also add " Story" to the name if we have decided to separate them from traditional CMs

Reply:
I personally am also not sold on it yet, it was weird seeing "Story Mode" and "Challenge Mode" for Mordremoth, but then we do not have a "Fractal Challenge Mode" or "Raid Challenge Mode", maybe StoryCM can just be removed and we'll see in future if changes are needed.

Reply:
That is also fine by me, Mordremoth is quite an exception, not the kind of things we generally add support for.